pkgs/development/python-modules/openai-triton/default.nix

```diff
@@ -224,6 +219,7 @@ buildPythonPackage rec {
     maintainers = with maintainers; [
```

Comment:
It appears that the repo got moved, please update the homepage to the new location.